pkg/roachprod/promhelperclient/client_test.go

```diff
@@ -57,8 +58,19 @@ func TestUpdatePrometheusTargets(t *testing.T) {
 			require.Equal(t, getUrl(promUrl, "c1"), url)
 			ir, err := getInstanceConfigRequest(io.NopCloser(body))
 			require.Nil(t, err)
-			// TODO (bhaskar): check for the correct yaml
 			require.NotNil(t, ir.Config)
+			configs := make([]*CCParams, 0)
+			require.Nil(t, yaml.UnmarshalStrict([]byte(ir.Config), &configs))
+			require.Equal(t, 2, len(configs))
+			for _, c := range configs {
+				if c.Labels["node"] == "1" {
+					require.Equal(t, "n1", c.Targets[0])
+				} else {
+					require.Equal(t, "n3", c.Targets[0])
+					require.Equal(t, "3", c.Labels["node"])
+				}
+				c.Labels["tenant"] = "system"
```

Comment:
We should be asserting that this is equal not assigning this right?

Reply:
Yes- thanks for pointing out. I actually fixed it locally and did not push it.